homeassistant/components/vesync/common.py

```diff
@@ -51,6 +109,8 @@ async def async_process_devices(hass, manager):
 class VeSyncBaseEntity(Entity):
     """Base class for VeSync Entity Representations."""
 
+    device: VeSyncBaseDevice
+
```

Comment:
Doesn't see useful.

Reply:
Possibly got brought in too early.  I have a whole suite of refactored code, this is probably more useful with a different set of changes.  I'll pull out